src/lib/data/history.test.ts

```diff
@@ -0,0 +1,28 @@
+import {startRide, stopRide} from '../ride';
+import {
+  updateRealTimeRecordRandom,
+  getOrCreateRealtimeRecord,
+  onSnapshotEvent,
+} from './realtime';
+import {generateTCX} from './history';
+import {TrainingCenterDatabase} from 'tcx-builder';
+
+it('generate a tcx with from ride history', async () => {
```

Comment:
I assume this probably should be `'generate a tcx from ride history'` removing the extra `with`